sips/minutes/_posts/2017-02-14-sip-minutes.md

```diff
@@ -0,0 +1,246 @@
+
+**Dmitry** If it implements a super-trait vals, traits still have setters, even for vals.
+
+**Sébastien** Yes, but that's only if it's mixed in. If it's overridden in the object, you don't need the setter. And if it's mixed in from the interface it doesn't have the static annotation anyway because that's now how ____ (15:20).
+
+**Martin** Does it even have setters anymore? In dotty it definitely doesn't. I thought in 2.12 they changed the scheme now as well for trait vals. I don't think they require setters anymore.
+
+**Dmitry** We don't require them but I'm not sure about Scalac. Does Scalac use initializers or trait setters?
+
+**Sébastien** Trait setters
+
+**Dmitry** For us, we can easily say that we support final but we're trying to take into account scalac. So is there some important use case where you want to have static fields implement not-static signatures.
+
+**Sébastien** No
+
+**Dmitry** Would you be willing to lean on a conservative way in this case?
+
+**Sébastien** I was just trying to understand why there is a restriction. There doesn't seem to be any reason to but if there is a reason to, I don't see a use case.
+
+**Dmitry** I would propose to record this but not change the SIP. To summarize in short, static is an annotation which does affect the compilation scheme which enforces some requirements where it can be which try to hide the fact that semantic would have changed by making it hard to observe this. The assumption is that most common users won't be able to observe the difference in semantics because we've restricted the syntax so that it would be hard to observe it. At the same time, for advanced users, it would allow to emit static fields and methods which will help if someone wants to write highly optimized code or interact with Java. I think the SIP is good as is. Given the fact ____ proposed some suggestions and clarifications about actual changes.
+
+**Jorge** The question is whether this should be accepted or not. The problem is we don't have any implementation for Scalac. Does someone at Lightbend plan to work on this sometime soon?
+
+**Seth** I don't think that's something we've discussed as a team yet.
+
+**Jorge** We have to pass here both on this proposal as is right now but I think this could be dangerous in the case where we don't have an implementation for Scalac because maybe the details change and assume something in Scalac that the SIP is not able to predict or guard against it. Let's wait until next month and I will double check whether this is possible or not. Then I will get in touch with the Lightbend team to see whether this can be implemented or not. We'll decide in a month whether it should be accepted.
+
+**Sébastien** ScalaJS already implemented it under another name but it's supposed to be conservative with respect to the aesthetic SIP in the sense that things that are allowed now with @jsstatic will also be allowed with @static. @static might open up a little bit more.
+
```

Comment:
Can we add a short "Conclusion" paragraph at the end of every discussion to summarize the decision that was taken? You can get inspiration from other minutes in the past :smile:.